src/styles/default.css

```diff
@@ -49,4 +49,13 @@ h3 {
 
 .u-widget {
   margin: 0 2em;
+  max-height: 17em;
+  overflow-y: auto;
 }
+
+.u-legend {
+  font-family: "Noto Serif";
```

Comment:
@jacobwegner so, this isn't possible as the way Michael set this up was not using sass but just a plain global stylesheet, so, no vars in scope here. I have been meaning to bring the widgets into line with `sv-mini` and use `style-resources-loader`, but I think in this case I should fold it into the next ticket/sprint rather than do it now?

Reply:
@jhrr I see; seems like a perfect candidate for a future ticket!  created as:

![](https://github.trello.services/images/mini-trello-icon.png) [Refactor default.css in scaife-widgets](https://trello.com/c/R07nQKXw)